packages/transmute-cli/src/commands/provision/index.js

```diff
@@ -1,107 +1,119 @@
 const run = require('../runner');
 const logger = require('../../logger');
-const TRANSMUTE_KUBE_VERSION = process.env.TRANSMUTE_KUBE_VERSION || 'v1.9.0';
+
+const TRANSMUTE_KUBE_VERSION = process.env.TRANSMUTE_KUBE_VERSION || 'v1.10.0';
 const MINIKUBE_CPU = process.env.MINIKUBE_CPU || '4';
 const MINIKUBE_MEMORY = process.env.MINIKUBE_MEMORY || '4096';
 const MINIKUBE_DISK = process.env.MINIKUBE_DISK || '100g';
 const MINIKUBE_PROFILE = process.env.MINIKUBE_PROFILE || 'transmute-k8s';
 const MINIKUBE_DRIVERS = ['virtualbox', 'kvm', 'kvm2', 'none'];
 
-module.exports.minikube = (dryrun, clusterName, minikubeDriver) => {
-    let minikube_param =
-        ' -e kubernetes_version=' +
-        TRANSMUTE_KUBE_VERSION +
-        ' -e minikube_disk_size=' +
-        MINIKUBE_DISK +
-        ' -e minikube_cpus=' +
-        MINIKUBE_CPU +
-        ' -e minikube_memory=' +
-        MINIKUBE_MEMORY;
-    +' -e minikube_profile=' + MINIKUBE_PROFILE;
+module.exports.minikube = (dryrun, minikubeDriver) => {
```

Comment:
From here until end of file, commit fixes indentation (4 spaces corrected to 2 spaces).